config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
-   namespace :v1 do 
+  namespace :v1 do 
+    mount_devise_token_auth_for 'User', at: 'auth'
     resources :investors, only: [:index, :show, :update, :create, :destroy]
-    resources :projects, only: [:index, :show, :update, :create, :destroy] do  collection do
-      get :my_projects
-    end
-  end 
+    resources :sessions, only: [:create, :destroy]
+    resources :projects, only: [:index, :show, :update, :create, :destroy] do  
+      collection do
+        get :my_projects
+      end
+    end 
     resources :users, only: [:index, :show, :update, :create, :destroy]
```

Comment:
I don't think you are using this controller. Delete

And check and delete for every endpoint that you have if it is used or not, if not used - delete.

Reply:
gud